pkgs/tools/security/ghidra/bin.nix

```
@@ -0,0 +1,70 @@
+{ stdenv, fetchzip, lib, makeWrapper, autoPatchelfHook
```

```suggestion
{ stdenv, fetchzip, makeWrapper, autoPatchelfHook
```

lib can be replaced with stdenv.lib.